roles/fstat/tasks/main.yml

```diff
@@ -1,17 +1,42 @@
 ---
+- user:
+    name: "{{user}}"
+
 - name: Install required packages
   package:
     state: present
     name: "{{ item }}"
   with_items:
-  - python-virtualenv
-  - gcc
-  - git
+    - python-virtualenv
+    - gcc
+    - git
 
-# TODO create the directory /fstat, do the git checkout
-# automate update
-# set configuration file, and database creation
+# TODO set configuration file, and database creation
 # and selinux boolean
+
+- name: create fstat directory
+  file:
+    path: /fstat/code state=directory
+
+- name: Check if virtualenv exists
+  stat:
+    path: /fstat/env
+  register: env
+
+- name: creates virtualenv if it does not exists
+  command: virtualenv /fstat/env
```

Comment:
Take a look at the `creates` parameter for the command module. That's what misc asked you to use. It makes the lines 21-24 and 28 redundant.